packages/multi-entry/src/index.js

```diff
@@ -2,7 +2,13 @@
 
 import { promise as matched } from 'matched';
 
-const entry = '\0rollup:plugin-multi-entry:entry-point';
+let entry;
+if (process.platform === "win32") {
+  // the colon is different
+	entry = '\0rollup꞉plugin-multi-entry꞉entry-point';
```

Comment:
this is an interesting hack (great thinking). @lukastaegert can you see a situation in which this would cause problems? if not, we should just unify on that unicode character for all platforms.

Reply:
@shellscape i am not sure how linux handels that unicode char at last inside bash i could expect that copy paste of that file name will result into not opening the file but it needs testing i never used such chars on linux filesystems before i think it is clever to do this only for windows so that nothing else breaks that already works.

Reply:
@shellscape i see a problem with the process.platform variable used inside browser should be no global  process at last.But in general a if condition is the way to go so we break not as much.

Reply:
The name of the final bundle is now `_rollup꞉plugin-multi-entry꞉entry-point.js`. Is this the desired behavior?

My question is why you don't give an option to the user for choosing the name?

Reply:
Ok, really stupid question, but there is no reason at all to have those colons, and using a clever character substitution that will even depend on the system running it does not feel like a good idea. So why not just a new name? And while we are at it considering that this is becomes a real file name when using the `dir`, why not just `entry.js` or `index.js`? I also think that there is no reason at all to use the `\0` prefix, on the contrary, it could be beneficial if a user could run Babel transforms on that file. And in the long run, it might make sense to make the actual name a config option of this plugin for the case when the `dir` option is used.

Reply:
I prefer to make it configurable. I prefer not to use `shx` to rename the file after rollup creates it.

Reply:
The name was never meant to be used for a file, this plugin predates the `dir` option. I would rather make a major release to fix this.

Reply:
@lukastaegert Any plan to fix this in a short time? Now a lot of windows developers can't use our library because of this issue, so if I know that it takes a lot of time to fix maybe I can patch this temporarily for ourselves or change our build config to remove the dependency on `multi-entry` plugin. (maybe we can replace this plugin with a glob and fs)

Reply:
Well, @frank-dspeed could just adapt the pull request. Not sure if he has plans to do so.

Reply:
So making it configurable definitely makes sense and should be added for the case when the `dir` options is used, but the old name is a very unfortunate default that should be changed as well. Would you add these changes?

Reply:
We would also need a test for that case and the new option, as well as a test using the `dir` option without specifying the entry name.

Reply:
This is just a patch. I say let's merge this as it is to fix the issue temporarily. In a separate pull request, we can make a feat to add options.

Reply:
@aminya I suggest using another clean name like `rollup_plugin_multi_entry_entry_point`

Reply:
Yes, that would be my requirement as well. If you think that would be breaking, then your patch would be breaking as well as it changes the name on Windows. But worse, it will make the windows build differ from what is generated on other systems.
In any case, even if the option is not added, a single test using the `dir` option would still be important.

Reply:
> @aminya I suggest using another clean name like rollup_plugin_multi_entry_entry_point

Please no. That's damn ugly.